src/config.rs

```diff
@@ -150,6 +154,10 @@ pub(crate) fn domain_name() -> String {
     dotenv::var("DOMAIN_NAME").unwrap_or_else(|_| "crates.io".into())
 }
 
+fn require_email_verification() -> bool {
+    dotenv::var("DISABLE_EMAIL_VERIFICATION_REQUIREMENT").is_err()
```

Comment:
if one had `DISABLE_EMAIL_VERIFICATION_REQUIREMENT=0`, won't this give a surprising result

Reply:
Yes, it would be surprising, but I think consistency with the other fields in the config is more important. There are a few others that follow this same pattern in this config, where it's just checking set or unset. For example, MIRROR and HEROKU.